app/views/toggle/_permanent_toggle_row.html.erb

```diff
@@ -1,5 +1,5 @@
 <% activated = @facade.activated_toggles.include?(toggle['identifier']) %>
```

Comment:
New partial for the permanent toggle row, as it now differs from a temporary toggle row without the updated_at status.